simulation_server/beamdriver.py

```diff
@@ -307,7 +307,14 @@ def update_pvs(self, pv_list):
         print("PVs updated.")
 
     def read(self, reason):
-        value = self.virtual_accelerator.get_pvs([reason])[reason]
+        try:
+            value = self.virtual_accelerator.get_pvs([reason])[reason]
+        except AttributeError as e:
+            if reason not in self.omitted:
+                self.omitted.append(reason)
+                print(f"Error getting param for {reason}: {e}, do not use {reason}")
+            return 0
```

Comment:
I think this would error out when `caget`-ing `SOMETHING.DRVL`. It might be worthwhile to ignore PVs with a `.XYZ` suffix.